src/interface/TWHash.cpp

```diff
@@ -6,86 +6,88 @@
 
 #include "../Data.h"
 
-#include <TrustWalletCore/TWHash.h>
+#include "BinaryCoding.h"
+#include "xxhash64.h"
```

Comment:
Should we name the file XXHash64.h (Pascal-case, as for other files).

Reply:
done